### PR TITLE
EWPP-2855: Logging actions and events.

### PR DESCRIPTION
**Pull request by upchuk**

### CODE REVIEW

File: modules/oe_translation_epoetry/modules/oe_translation_epoetry_mock/src/Logger/MockLogger.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<?php
+
```
Comment (22Alexandra) [inline modules/oe_translation_epoetry/modules/oe_translation_epoetry_mock/src/Logger/MockLogger.php:2]:
```suggestion

declare(strict_types = 1);

```

**Review by 22Alexandra — COMMENTED** [review]
Not related to the PR.

In `oe_translation_corporate_workflow_node_presave` we still assume that every entity has a `moderation_state` field so translations cannot be synced for entities without it.

**Comment by upchuk** [comment]
@22Alexandra thanks! fixed in https://github.com/openeuropa/oe_translation/pull/199/files#diff-26302cc749332e2967d815e5ac652200754de8089b1fcfd03e3f5dcabac2d3af

**Review by hernani — APPROVED** [review]
Minor comments

File: modules/oe_translation_epoetry/src/EventSubscriber/NotificationsSubscriber.php [hunk 2]
```diff
@@ -369,4 +399,62 @@ protected function getTranslationRequest(RequestReference $reference): ?Translat
...
+  /**
+   * Formats the request reference into a readable string.
+   *
+   * @param \OpenEuropa\EPoetry\Notification\Type\RequestReference $reference
+   *   The reference.
+   *
+   * @return string
+   *   The string reference.
+   */
+  protected function formatRequestReference(RequestReference $reference): string {
```
[the remark this answers is not in the document]
Comment (upchuk) [reply modules/oe_translation_epoetry/src/EventSubscriber/NotificationsSubscriber.php:411]: True, but for now, the library RequestReference::getReference() wraps the version into () for some reason. Have to see why. We might change it and then we use that method.

**Pull request merged**